fastlane/Helpers/Build.swift

```diff
@@ -51,7 +51,8 @@ enum Build {
             ]),
             buildPath: .userDefined(Constant.buildPath),
             derivedDataPath: .userDefined(Constant.derivedDataPath),
-            xcodebuildFormatter: "xcbeautify"
+            xcodebuildFormatter: "xcbeautify",
+            clonedSourcePackagesPath: .userDefined(Constant.clonedSourcePackagesPath)
```

Comment:
- For `disablePackageAutomaticUpdates`. If we set it to `true`, it will be triggered updating packages every time even there's nothing changes. Therefore, I think we can safly leave it with the default value (false). 🙏 

[This job](https://github.com/nimblehq/nimble-crypto-ios/actions/runs/3779751588/jobs/6425264936) I set `disablePackageAutomaticUpdates` `true` and running time is more than when we set it to `false`.

![Screenshot 2022-12-26 at 14 22 30](https://user-images.githubusercontent.com/25881847/209517561-0adbcc70-4843-4fa2-9e36-c39503d12fe0.png)

- For `skipPackageDependenciesResolution`. If we set it to `true`, it will skip this part:

![Screenshot 2022-12-26 at 14 24 18](https://user-images.githubusercontent.com/25881847/209517970-09fcc0a0-95ee-43d0-85ba-7fb59ecc06cb.png)

- The speed comparison is almost the same.
![Screenshot 2022-12-26 at 14 23 01](https://user-images.githubusercontent.com/25881847/209517638-4b194bc9-3d06-421e-8a7e-0a2f355e9ba7.png)

=> [This job](https://github.com/nimblehq/nimble-crypto-ios/actions/runs/3779580561) with `skipPackageDependenciesResolution`.
=> [This job](https://github.com/nimblehq/nimble-crypto-ios/actions/runs/3779525632/jobs/6424854293) without `skipPackageDependenciesResolution`.

Because of that, I leave it as default (false).
Please correct me if I'm wrong 🙏 .

Reply:
Since our project has few dependencies, the `skipPackageDependenciesResolution`'s benefit might not be clear, and its impact is not much. Keep its value as default (the `disablePackageAutomaticUpdates` as well) is fine to me🙏